setup.py

```diff
@@ -5,7 +5,7 @@
     version='1.1',
     url='https://github.com/podshumok/python-smpplib',
     description='SMPP library for python',
-    long_description=open('README.md', 'rt').read(),
+    long_description=open('README.md', 'rt', encoding='utf-8').read(),
```

Comment:
Oops, I was too fast to approve. `encoding` is indeed missing in Python 2. I'll change it to `io.open`, if you don't mind.